### ISSUE TITLE
TOML file path output between every test

### ISSUE THREAD

**Issue by whoabuddy** [hirosystems/clarinet#320]
I noticed this first in GitHub actions, then locally after upgrading from `0.24.0` to `0.28.0`. Some examples below:

[From the GitHub Action:](https://github.com/citycoins/citycoin/runs/6030803576?check_suite_focus=true)

```
Running file:///github/workspace/tests/citycoin-token.test.ts
"/github/workspace/Clarinet.toml"
* [CityCoin Token] SIP-010 FUNCTIONS transfer() succeeds with no memo supplied ... ok (44ms)
"/github/workspace/Clarinet.toml"
* [CityCoin Token] SIP-010 FUNCTIONS transfer() succeeds with memo supplied ... ok (80ms)
"/github/workspace/Clarinet.toml"
* [CityCoin Token] SIP-010 FUNCTIONS transfer() fails with u1 when sender does not have enough funds ... ok (31ms)
"/github/workspace/Clarinet.toml"
* [CityCoin Token] SIP-010 FUNCTIONS transfer() fails with u2 when sender and recipient are the same ... ok (37ms)
"/github/workspace/Clarinet.toml"
```

From my local build:

```
Running file:///home/whoabuddy/Dev/citycoins/contracts/tests/citycoin-vote-v1.test.ts
"/home/whoabuddy/Dev/citycoins/contracts/Clarinet.toml"
Running file:///home/whoabuddy/Dev/citycoins/contracts/tests/citycoin-token.test.ts
"/home/whoabuddy/Dev/citycoins/contracts/Clarinet.toml"
* [CityCoin Vote] VOTE ACTIONS intialize-contract() fails with ERR_UNAUTHORIZED if the contract is already initialized ... ok (15ms)
"/home/whoabuddy/Dev/citycoins/contracts/Clarinet.toml"
* [CityCoin Token] SIP-010 FUNCTIONS transfer() succeeds with no memo supplied ... ok (20ms)
"/home/whoabuddy/Dev/citycoins/contracts/Clarinet.toml"
* [CityCoin Vote] VOTE ACTIONS intialize-contract() fails with ERR_UNAUTHORIZED if the sender is not the deployer ... ok (13ms)
"/home/whoabuddy/Dev/citycoins/contracts/Clarinet.toml"
* [CityCoin Token] SIP-010 FUNCTIONS transfer() succeeds with memo supplied ... ok (19ms)
"/home/whoabuddy/Dev/citycoins/contracts/Clarinet.toml"
* [CityCoin Vote] VOTE ACTIONS intialize-contract() fails with ERR_UNAUTHORIZED if the start height is before the current height ... ok (30ms)
"/home/whoabuddy/Dev/citycoins/contracts/Clarinet.toml"
```

Everything is running correctly but this definitely makes it hard to read!

**Comment by lgalabru** [comment]
Good catch, thanks - to be addressed with #323

**Comment by whoabuddy** [comment]
Confirming this is fixed in v0.29.0 :clap: :pray:
